app/screens/home/search/search.tsx

```diff
@@ -130,11 +133,12 @@ const SearchScreen = ({teamId}: Props) => {
         };
     }, [headerHeight, lastSearchedValue]);
 
-    const onClear = useCallback(() => {
+    const handleClearSearch = useCallback(() => {
         setSearchValue('');
         setLastSearchedValue('');
         setFilter(FileFilters.ALL);
-    }, [filter]);
+        setShowResults(false);
+    }, [filter, searchValue, lastSearchedValue]);
```

Comment:
Not sure why this used to have the `filter` as a dependency, but seems like it depends on nothing.

Reply:
@larkox, shouldn't it be dependent on `[setSearchValue, setLastSearchedValue, setFilter, setShowResults]`?